plugins/dynamix/SysDevs.page

```diff
@@ -14,53 +14,48 @@ Tag="list"
  * all copies or substantial portions of the Software.
  */
 ?>
+<style>
+table.pre{margin-top:0}
+table.pre td:first-child{width:144px}
+table tr td{padding:0 0 3px 0;margin:0}
+table tr td.thin{line-height:8px;height:8px}
+</style>
+<script>
+$(function(){
+  $.get('/webGui/include/SysDevs.php',{table:'t1'},function(data){$('#t1').html(data)});
```

Comment:
A neat shortcut for each of these $.get lines:
```$('#t1').load('/webGui/include/SysDevs.php',{table:'t1'});```

Reply:
Nifty :)